mlflow/langchain/__init__.py

```diff
@@ -1017,71 +1016,78 @@ def autolog(
             MlflowLangchainTracer as a callback during inference. If ``False``, no traces are
             collected during inference. Default to ``True``.
     """
-    with contextlib.suppress(ImportError):
```

Comment:
note: Github diff goes crazy, but I just removesd`with contextlib.suppress(ImportError)` (so we don't fail silently)